### ISSUE TITLE
PowerShell does not accept paths with spaces in -File argument

### ISSUE THREAD

**Issue by orybak** [PowerShell/PowerShell#12186]
# Steps to reproduce
 - Create a folder with space in it (for example `my folder`)
 - Create a simple script inside this folder (`script.ps1`)
```powershell
Write-Output "Hello, World"
```
- run with command `pwsh -File "/Users/username/my folder/script.ps1" `

# Expected behavior

Script executes successfully:
```none
Hello, World
```
Note: script does execute correctly this way if run under PowerShell Desktop under Windows

# Actual behavior

```none
➜ /Users/username pwsh -File "/Users/username/my folder/script.ps1"
The argument 'my' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Usage: pwsh[.exe] [-Login] [[-File] <filePath> [args]]
                  [-Command { - | <script-block> [-args <arg-array>]
                                | <string> [<CommandParameters>] } ]
```

Same under Windows and PowerShell 6.2.4

```none
C:\work\src>pwsh -File "my folder\script.ps1"  
The argument 'my' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.  
```

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Darwin 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_…
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
Windows part

```none
Name                           Value
----                           -----
PSVersion                      6.2.4
PSEdition                      Core
GitCommitId                    6.2.4
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0


```

**Comment by mklement0** [comment]
It sounds like you installed PowerShell as .NET global tool (`dotnet tool install powershell -g`)

Sadly, a PowerShell version installed this way uses a shim executable whose argument parsing is fundamentally broken  - please see #11747

By contrast, a regularly installed PowerShell 7  version isn't affected.

**Comment by orybak** [comment]
@mklement0 yes, that's indeed how I installed it. This issue can be closed as a duplicate then. 
Thank you

**Comment by daxian-dbw** [comment]
Closed as a duplicate of #11747
